docs/source/kedro-viz_visualisation.md

```diff
@@ -30,6 +30,10 @@ The next step is optional, but useful to check that all is working. Run the full
 kedro run
 ```
 
+```{note}
```

Comment:
We don't mention dataset stats feature anywhere in the docs, maybe we first give an idea what it is and then say from Kedro-viz 9.2.0 stats are saved in .viz folder? something like below? 


Kedro-Viz provides stats related to datasets under the metadata panel

 (insert image). 
 
 These stats are generated by hooks. If you have Kedro-Viz installed and run kedro run, these hooks will save the stats by default. To disable this, you can disable Kedro-Viz hooks in the settings. Starting from version 9.2.0, Kedro-Viz stats are saved in the .viz folder.